### PR TITLE
Feature/radius configurator

### PR DESCRIPTION
**Pull request by whitx**
# Description
Allow the configurator to be configured as RADIUS only

# Impacts
Configurator flow

# Delete branch after merge
YES

# NEWS file entries
## New Features
* Added a RADIUS only mode to PacketFence.

### CODE REVIEW

**Comment by whitx** [comment]
Should we changed which services are running for this mode?
Considering portal, dhcp, dns etc will not be used as there is no isolation/registration VLAN

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -64,6 +64,9 @@ Hotspot support (Web Auth Enforcement)::
   PacketFence can also be configured as hotspot, if you have a manageable device that supports an external captive portal (like
 Cisco WLC or Aruba IAP).
 
+RADIUS support (RADIUS Enforcement)::
+  PacketFence can act as RADIUS only server, without having a registration, isolation, portal interface. In case you want to use PacketFence for your 802.1x authentication only.
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: 802.1x -> 802.1X

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -456,6 +459,16 @@ An example:
 
 This will trigger all the nodes that connects to the _GuestAccess_ SSID to use inline enforcement mode (PacketFence will return a void VLAN or the `inlineVlan` if defined in switch configuration) and the MAC address `00:11:22:33:44:55` client if it connects on another SSID.
 
+Technical introduction to RADIUS enforcement
+--------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+The concept of having a RADIUS enforcement is to not use registration and isolation, nor the portal. Everything here is for 802.1x authentication only. By default your management interface will be your RADIUS interface, if needed you can add another one via 'Network Configuration->Networks->Interfaces' adding an `Other` type of interface and the additionnal daemon `RADIUS`.
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc]: 802.1x -> 802.1X

**Comment by extrafu** [comment]
👍

File: html/pfappserver/lib/pfappserver/Model/Enforcement.pm [hunk 3]
```diff
@@ -27,6 +27,7 @@ my %types   = (
 #    option      => [ 'high-availability', 'dhcp-listener', 'monitor' ],
     webauth     => ['management', 'portal'],
     other       => ['dns-enforcement'],
+    radius      => ['management'],
```
Comment (dwlfrth) [inline html/pfappserver/lib/pfappserver/Model/Enforcement.pm:30]: I think you are missing "radius" here ;)

Comment (whitx) [reply html/pfappserver/lib/pfappserver/Model/Enforcement.pm:30]: This is actually the 'required' types, since management contain radius it should not be necessary.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -64,6 +64,9 @@ Hotspot support (Web Auth Enforcement)::
   PacketFence can also be configured as hotspot, if you have a manageable device that supports an external captive portal (like
 Cisco WLC or Aruba IAP).
 
+RADIUS support (RADIUS Enforcement)::
+  PacketFence can act as RADIUS only server, without having a registration, isolation, portal interface. In case you want to use PacketFence for your 802.1X authentication only.
```
Comment (dwlfrth) [inline docs/PacketFence_Administration_Guide.asciidoc]: I'd rephrase that to be more "general", to be future proof.
"PacketFence can be used as a RADIUS server without having registration, isolation, portal features in the case only RADIUS authentication for network equipment is what matter."

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 5]
```diff
@@ -456,6 +459,16 @@ An example:
 
 This will trigger all the nodes that connects to the _GuestAccess_ SSID to use inline enforcement mode (PacketFence will return a void VLAN or the `inlineVlan` if defined in switch configuration) and the MAC address `00:11:22:33:44:55` client if it connects on another SSID.
 
+Technical introduction to RADIUS enforcement
+--------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+The concept of having a RADIUS enforcement is to not use registration and isolation, nor the portal. Everything here is for 802.1X authentication only. By default your management interface will be your RADIUS interface, if needed you can add another one via 'Network Configuration->Networks->Interfaces' adding an `Other` type of interface and the additionnal daemon `RADIUS`.
```
Comment (dwlfrth) [inline docs/PacketFence_Administration_Guide.asciidoc]: The concept of having a RADIUS enforcement is to not use registration and isolation, nor the portal. Everything here is for RADIUS authentication only. By default the management interface will be the RADIUS interface. If needed, it is possible to add another one via 'Network Configuration->Networks->Interfaces' adding an `Other` type of interface and the additional daemon `RADIUS`.

**Review by dwlfrth — COMMENTED** [review]
Rephrasing doc

**Comment by dwlfrth** [comment]
That's weird
screen shot 2017-08-18 at 11 41 31

**Comment by dwlfrth** [comment]
Other than the documentation modification, LGTM
